src/components/footer/styles.js

```diff
@@ -0,0 +1,33 @@
+import styled, { css } from 'styled-components';
```

Comment:
The layout doesn't match the design yet. `justify-content: space-between` is not working in this case. You can use `flex: 1` inside the `LinkStyles`

![image](https://user-images.githubusercontent.com/9722792/77855850-77419600-71f3-11ea-8e75-0bce028846b5.png)